.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
   - repo: https://github.com/ansible-network/collection_prep
     rev: 1.1.1
     hooks:
-      - id: autoversion
+      # - id: autoversion # removed as being handled by GHA push and release drafter
```

Comment:
You can just delete it it's fine. It's not hard to restore if we decide to go back to it